setup.py

```diff
@@ -55,18 +55,18 @@ def parse_requirements_file(path):
         ':sys_platform=="win32"': ["colorama"],
         "docs": parse_requirements_file("doc_requirements.txt"),
     },
-    python_requires=">=3.8.0,<3.13",
+    python_requires=">=3.9.0,<3.14",
```

Comment:
3.13 isn't out yet, please revert the upper bound change

Reply:
That'd clash with hikari's range again, which was the reason for this PR in the first place. At this point, I'd rather wait until 3.13 releases next week tbh

Reply:
With Python 3.13 now released, I consider this resolved